scripts/enum/pet_id.lua

```diff
@@ -57,6 +57,26 @@ xi.petId =
     PANZER_GALAHAD        = 45,
     CHOPSUEY_CHUCKY       = 46,
     AMIGO_SABOTENDER      = 47,
+    CRAFTY_CLYVONNE       = 49,
+    BLOODCLAW_SHASRA      = 50,
+    LUCKY_LULUSH          = 51,
+    FATSO_FARGANN         = 52,
+    DISCREET_LOUISE       = 53,
+    SWIFT_SIEGHARD        = 54,
+    DIPPER_YULY           = 55,
+    FLOWERPOT_MERLE       = 56,
+    NURSERY_NAZUNA        = 57,
+    MAILBUSTER_CETAS      = 58,
+    AUDACIOUS_ANNA        = 59,
+    PRESTO_JULIO          = 60,
+    BUGEYED_BRONCHA       = 61,
+    GOOEY_GERARD          = 62,
+    GOREFANG_HOBS         = 63,
+    FAITHFUL_FALCOR       = 64,
+    CRUDE_RAPHIE          = 65,
+    DAPPER_MAC            = 66,
+    SLIPPERY_SILAS        = 67,
+    TRUBID_TOLOI          = 68,
```

Comment:
Typo here - this should be `TURBID_TOLOI`.

Reply:
Good catch, thanks

Reply:
Thanks Almuth, you're the best.